ansible/playbooks/roles/containerd/tasks/main.yml

```diff
@@ -0,0 +1,67 @@
+---
+- name: Remove Docker
+  block:
+  - name: Stop Kubelet before Docker removal
+    systemd:
+      name: kubelet
+      state: stopped
+    notify:
+    - Restart kubelet
+
+  - name: Remove Docker
+    include_role:
+      name: docker
+      tasks_from: remove-docker.yml
+  when:
+    - is_upgrade_run
+    - inventory_hostname not in groups.image_registry
+
+- name: Install Containerd package
+  package:
+    name: containerd.io
```

Comment:
We may have more than one package version in epirepo (e.g. containerd.io is also docker-ce dependency) so maybe it would be good to specify version here as well - like we have for Docker packages.